pkg/eden/eden.go

```diff
@@ -353,6 +353,46 @@ func StatusEVEQemu(pidFile string) (status string, err error) {
 	return utils.StatusCommandWithPid(pidFile)
 }
 
+//StartEVEParallels function run EVE in parallels
+func StartEVEParallels(vmName string, parallelsCpus int, parallelsMem int) (err error) {
+	commandArgsString := fmt.Sprintf("create %s --distribution ubuntu --no-hdd", vmName)
+	if err = utils.RunCommandWithLogAndWait("prlctl", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("prlctl error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("set %s --device-del net0", vmName)
+	if err = utils.RunCommandWithLogAndWait("prlctl", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("prlctl error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("set %s --device-add hdd --image dist/default-images/eve/live.parallels --cpus %d --memsize %d", vmName, parallelsCpus, parallelsMem)
+	if err = utils.RunCommandWithLogAndWait("prlctl", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("prlctl error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("set %s --device-add net --type shared --adapter-type virtio --ipadd 192.168.1.0/24", vmName)
+	if err = utils.RunCommandWithLogAndWait("prlctl", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("prlctl error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("set %s --device-add net --type shared --adapter-type virtio --ipadd 192.168.2.0/24", vmName)
+	if err = utils.RunCommandWithLogAndWait("prlctl", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+		log.Fatalf("prlctl error for command %s %s", commandArgsString, err)
+	}
+	commandArgsString = fmt.Sprintf("net set Shared --nat-tcp-add 2222_22,2222,%s,22", vmName)
```

Comment:
We need to use values from our config here. Can you please pass it the same way as [here](https://github.com/lf-edge/eden/blob/master/cmd/eve.go#L81-L83).